### ISSUE TITLE
q&V: MKAP query request

### ISSUE THREAD

**Issue by luizaogs** [arch-kiosk/arch-kiosk-office#2635]
Going through MKAP Kiosk with Kelly to see what we might need for this publication, and a query that would be useful is something similar to the "What materials (and how much of each) were found in unit X?" but showing what material is a small find and what is a bulk, etc.

Would that be possible?

(It also seems like the end goal is some sort of finds table for the article, so you might get a request for a report.)

**Comment by urapadmin** [comment]
ja, totally. 🥳

**Comment by urapadmin** [comment]
### 06.V.2024
- the query now has the collected material type as another column
- it also has pie charts for both cm type and material

**Comment by urapadmin** [comment]
what one would really want is a stacked bar chart with the collected material types as the main category and the materials stacked within that or the other way around. But that would need a stacked bar chart as an option in Q&V first.
I make a new ticket for that. Closing one, opening one. Arg.

**Comment by luizaogs** [comment]
You haven't rolled this out, right? I saw the date up there rather than a Kiosk version so wasn't sure, but I don't see these updates yet.

**Comment by urapadmin** [comment]
no, I found a bug before I could.

**Comment by urapadmin** [comment]
### kiosk 1.5.3.33
- this depends on #2698 first

**Comment by luizaogs** [comment]
You haven't rolled this out to MKAP (or at least Kiosk tells me you haven't) but I looked at this in PASU and yes, I believe that is what Kelly wanted. Looks good to me.

**Comment by urapadmin** [comment]
I keep confusing MKAP and MADP. It is now rolled out for MKAP, too.
